### PR TITLE
Links "LawTesting.md" in FAQ and TypeClasses pages

### LINKED ISSUE #1962: link lawtesting document somewhere
https://github.com/typelevel/cats/blob/master/docs/src/main/tut/typeclasses/lawtesting.md
can be reached at https://typelevel.org/cats/typeclasses/lawtesting.html
but it's not linked anywhere. We should probably link it from the [type classes](https://typelevel.org/cats/typeclasses.html) page in the law section as well as a FAQ item.

### PR DESCRIPTION
**Pull request by AlejandroME**
Fixes #1962.

### CODE REVIEW

File: docs/src/main/tut/faq.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ position: 40
  * [What does `macro Ops` do? What is `cats.macros.Ops`?](#machinist)
  * [What is `tailRecM`?](#tailrecm)
  * [What does this symbol mean?](#symbol)
+ * [How can I test `laws`?](#law-testing)
```
Comment (kailuowang) [inline docs/src/main/tut/faq.md]: A more specific way to say could be "how can I test instances against their type classes'  laws? "

Comment (AlejandroME) [reply docs/src/main/tut/faq.md]: Perfect!

File: docs/src/main/tut/typeclasses/typeclasses.md [hunk 2]
```diff
@@ -221,6 +221,8 @@ val result = Monoid[Int].combine(sumLeft, sumRight)
 Cats provides laws for type classes via the `kernel-laws` and `laws` modules which makes law checking
 type class instances easy.
 
+You can find out more about law testing [here](typeclasses/lawtesting.html).
```
Comment (kailuowang) [inline docs/src/main/tut/typeclasses/typeclasses.md]: Sorry I should've caught this earlier. This link probably has an extra typeclasses in path?

Comment (AlejandroME) [reply docs/src/main/tut/typeclasses/typeclasses.md]: @kailuowang The path is good. I've tested it.
Without the `typeclasses` prefix the URL formed is: /cats/lawtesting, but the original .md is placed in typeclasses/lawtesting.md.

Comment (kailuowang) [reply docs/src/main/tut/typeclasses/typeclasses.md]: oh cool, thanks for testing it.

**Comment by kailuowang** [comment]
Thanks very much @AlejandroME  for this nice fix.

**Pull request merged**